plugins/powerpaste.md

```diff
@@ -295,3 +295,39 @@ tinymce.init({
 The PowerPaste plugin provides the following JavaScript command.
 
 {% include commands/powerpaste-cmds.md %}
+
+{% include misc/support-powerpaste.md %}
+
+## Common questions and troubleshooting PowerPaste behavior
+
+### What happens when copy and pasting from Microsoft Word?
+
+When content is copied from an application (such as Microsoft Word), the application places an HTML representation of the copied content onto the computer's clipboard. PowerPaste uses the HTML from the clipboard and cannot access the file directly.
+
+> **Note**: Web browsers and the applications running in them cannot directly access files on the computer for security reasons.
+
+Microsoft Word or Microsoft Excel can create content that does not have equivalent HTML. The HTML provided to the clipboard by the application is the application's "best effort" at representing the content as HTML. Depending on the complexity of the source document, the content pasted into {{site.productname}} using PowerPaste may not be an exact representation of what the content looked like in the original application.
```

Comment:
"equivalent HTML" reads badly somehow to me, but I can't think of anything better.

Reply:
```suggestion
Microsoft Word or Microsoft Excel can create content that does not have an equivalent in HTML. The HTML provided to the clipboard by the application is the application's "best effort" at representing the content as HTML. Depending on the complexity of the source document, the content pasted into {{site.productname}} using PowerPaste may not be an exact representation of what the content looked like in the original application.
```
maybe?